routes/opportunity.js

```diff
@@ -84,6 +84,10 @@ const opportunityManager = new OpportunityManager(),
  *                         type: string
  *                         description: Link for the opportunity.
  *                         example: www.rohithmsrscholarship.com
+ *                       onlyForFemale:
+ *                         type: bool
```

Comment:
It should be `boolean`, check swagger doc

Reply:
Done!